### PR TITLE
[PFD] Add altitude indicator alerts

### PR DESCRIPTION
**Pull request by ilyeshammadi**
Partially related #1227

## Summary of Changes

I added animation on the altitude cursor on the PFD start flashing when the selected altitude is almost reached.

### What has been done:
Flash yellow or pulse amber based on the FCOM.

### What is not included:
The audio warnings are not included in this PR.

## Screenshots (if necessary)

ezgif com-gif-maker (2)
Video: https://www.youtube.com/watch?v=HKIbBS0a-Hk

## References

Yellow flashing: https://www.youtube.com/watch?v=TEi-WX08ov8
Amber pulsing: https://youtu.be/0LmVp6LCW8o?t=66
---------------------------------------------------------------------------------------------------------------------
95640895-5d84e380-0a9f-11eb-8dfe-4c49e558872d

## Additional context

Discord username (if different from GitHub): Union Araken#4083

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link

### CODE REVIEW

**Comment by Benjozork** [comment]
Hello, your first reference is good but FYI the second one models an old CRT screen, which we cannot use for reference.

**Comment by ilyeshammadi** [comment]
> Hello, your first reference is good but FYI the second one shows an old CRT screen, which we cannot use for reference.

Sure, let's keep just video then.

**Comment by beheh** [comment]
Nice change! This should not be pulsing though, but rather flashing (with no transition). I have [another reference video here](https://www.youtube.com/watch?v=b7YfxSxJWzs#t=30s), at 30s. From your original reference it's hard to tell due to the time lapse, but if you step through it frame by frame you can see it flashing too (the 2 frame fade is a time lapse artifact I presume).

**Comment by ilyeshammadi** [comment]
@beheh thank you so much for the references 😄

**Comment by beheh** [comment]
I've found another reference over at #1227, this time for the general altitude alert. The following diagram encodes both aural alerts/chimes and the visual PFD flashing you're implementing here:

grafik

It confirms the 200ft and 750ft thresholds. One thing that's still missing is the alert cancellation:
* [x] Turning FCU (works I believe, as that's what you reference against)
* [ ] L/G downlocked and Flaps > 25deg
* [ ] On Glideslope

...which make sense, as the pilots don't need to be warned about leaving the altitude when they're about to land.

It also shows there's another mode apart from the "yellow pulsing" that you've implemented here, namely the "amber flashing". Presumably the color is slightly different and it flashes with higher intensity. I'm trying to find a source for this, if you also want to pick that up in this PR.

**Comment by ilyeshammadi** [comment]
@beheh thanks for this, I agree we should include the Yellow/Amber behavior. I've changed the PR title and will focus on this. The next step may be todo in another PR will be to do the audio part like mentioned in #1227.
95640895-5d84e380-0a9f-11eb-8dfe-4c49e558872d

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/A320_Neo_PFD.css [hunk 1]
```diff
@@ -446,3 +446,33 @@ a320-neo-pfd-element #Mainframe #InstrumentsContainer #ILS {
   height: 100%;
   z-index: 1;
 }
+
+
```
Comment (Benjozork) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/A320_Neo_PFD.css:450]:
```suggestion
```

**Review by Benjozork — APPROVED** [review]
LGTM, can you just remove those empty lines ?

**Comment by FlightSimmer7700** [comment]
Category: QA Tester
Name: FlightSimmer7700#3468
Date of testing: Current date 16/11/20
Version of the sim: 1.10.11.0
PR Tested: #1889 
Object of test: PFD Altitude
Testing technique: Did a whole flight
Steps you've made to test the Pull Request:
1. Flew a flight
2. looked closely at when I was approaching the set altitude
3. It started flashing about 750ft before
4.
5.

Flight notes (If required):

Issues (If Any): 
Overall Rating: Good
Conclusions of the testing: Looks all good

**Comment by camilo625** [comment]
Category: QA Tester
Name: Camilo625 #5575
Date of testing: Current date 16/11/2020
Version of the sim: 1.10.11.0
PR Tested: #1889 
Object of test: PFD

Testing technique: Check during flight 
Steps you've made to test the Pull Request:
1. Spawn C&D
2. Start a flight
3. Check PFD
4. Land

Flight notes (If required): None.

Issues (If Any): None.
Overall Rating: Good
Conclusions of the testing: Works as described, nice work.

**Pull request merged**